.circleci/config.yml

```diff
@@ -28,8 +28,7 @@ commands:
             . /opt/conda/etc/profile.d/conda.sh
             conda activate esmvaltool
             flake8 -j 4
-            pytest -n 4 -m "not sequential" --junitxml=report-1.xml
-            pytest -n 0 -m "sequential" --cov-append --junitxml=report-2.xml
+            pytest -n 4 --junitxml=report-1.xml
             esmvaltool version
       - run:
           name: Merge test reports
```

Comment:
there is no need to merge test reports anymore if there is only one, so this could also be removed (make sure to use the correct name for the report on line 31 though).

Reply:
@bouweandela would you be a pie and do those changes yourself (wait til this batch of tests finishes, I am counting how many test batches I run) please - not really understanding what all that junit thing is all about :stuck_out_tongue_closed_eyes:

Reply:
Those xml files are coverage reports, they are collected during testing and then uploaded to codecov

Reply:
@bouweandela I fixed it now, can you have a look pls, man? :beer:

Reply:
Sorry, that explanation wasn't right: these test reports are used to generate tests results on CircleCI. It's coverage.xml that is used for test coverage. You can see an example of a working test report from the main branch here: https://app.circleci.com/pipelines/github/ESMValGroup/ESMValCore/7902/workflows/7fe28b7b-b8e7-484d-ba83-6b8b1461e450/jobs/36133/tests
and a failed one from this branch here:
https://app.circleci.com/pipelines/github/ESMValGroup/ESMValCore/7909/workflows/e2c09be1-836d-47c2-8602-a27bad51da5f/jobs/36152/tests

Reply:
yes I know about coverage.xml since I set up cov someplace else recently, but thought it takes reports from multiple sources, Thanks, man! You reckon we can merge this? No trace of any SegFaults from running a hefty number of tests (am just gonna let the last batch finish then I'll remove the calls to GA testing)